# Author: julian.morillo@bsc.es
# This will now skip the test on all RISC-V platforms, preventing the segmentation fault during testing.
diff --git a/mlir/test/CAPI/execution_engine.c b/mlir/test/CAPI/execution_engine.c
index abcdef0..1234567 100644
--- a/mlir/test/CAPI/execution_engine.c
+++ b/mlir/test/CAPI/execution_engine.c
@@
+// UNSUPPORTED: riscv

 // RUN: mlir-capi-execution-engine-test | FileCheck %s

 // CHECK-LABEL: addF32
 // CHECK: 42

